assets/js/page-enhancements.js

```diff
@@ -8,31 +8,6 @@
  * No code should be written here that renders a page non-functional if JS is not enabled or the script does not run/fails.
  */
 
-;(autoFocusBarcodeField = ($, document) => {
-  $(document).ready(() => {
-    $('#scan-barcode-form #barcode').focus()
-  })
```

Comment:
Moved all the autofocus stuff to the bottom - so the other enhancements that change page layout have a chance to do their thing first.